README.md

```diff
@@ -0,0 +1,60 @@
+# Shavar list testing using test pages
+Instructions on how to create test profiles: https://wiki.mozilla.org/Services/TrackingProtection/Shavar_Server_-_Testing#SET_CUSTOM_PREFS
```

Comment:
This is probably best for readability, but I wonder if relying on this as a source of truth as well as prefs.ini as a source of truth will be problematic maintaining both in the long run

Reply:
That may be a valid concern for the long run, however I don't see it changing anytime soon. Test rail works for details on setting up lists, but the wiki still has the larger picture of the project.